packages/account/src/Sections/Security/TwoFactorAuthentication/__tests__/two-factor-disabled.spec.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { APIProvider } from '@deriv/api';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import TwoFactorDisabled from '../two-factor-disabled';
+
+jest.mock('@deriv/components', () => {
```

Comment:
the PR's scope doesn't include test coverage. If it's included, it should be stated in PR's (and you card's) description. Normally, TS migration is not united with Test coverage in order to not overload PRs.

Reply:
A note has been added in the card description informing that changes including code movements across files and unit test coverages too are included in this PR. 🙏🏼